### PR TITLE
some example docs updates

### PR DESCRIPTION
**Pull request by yaleman**
Relates to #187

- [x] cargo fmt has been run
- [x] cargo test has been run and passes
- [x] documentation has been updated with relevant examples (if relevant)

### CODE REVIEW

File: webauthn-rs/src/lib.rs [hunk 1]
```diff
@@ -562,10 +563,10 @@ impl Webauthn {
     ///
     /// This function returns a `CreationChallengeResponse` which you must serialise to json and
     /// send to the user agent (e.g. a browser) for it to conduct the registration. You must persist
-    /// on the server the `SecurityKeyRegistration` which contains the state of this registration
+    /// on the server the [SecurityKeyRegistration] which contains the state of this registration
     /// attempt and is paired to the `CreationChallengeResponse`.
     ///
-    /// WARNING ⚠️  YOU MUST STORE THE `SecurityKeyRegistration` VALUE SERVER SIDE.
+    /// WARNING ⚠️  YOU MUST STORE THE [SecurityKeyRegistration] VALUE SERVER SIDE.
```
Comment (Firstyear) [inline webauthn-rs/src/lib.rs:569]: There is more than just the SecurityKeyRegistration type, isn't there also Passkey and PasswordlessKey?

**Review by Firstyear — CHANGES_REQUESTED** [review]
Looks really good, but doesn't this need to include the other registration types?

File: webauthn-rs/src/lib.rs (hunk 1 above)
Comment (yaleman) [reply webauthn-rs/src/lib.rs:569]: Yep, but I'm pretty clueless on that front, just did some quick changes to get the work started and show how it's done. 😄

Comment (Firstyear) [reply webauthn-rs/src/lib.rs:569]: I'll do the rest then :)

**Pull request merged**